source/develop/release-management/features/storage/vm-leases.html.md

```diff
@@ -0,0 +1,292 @@
+---
+title: VM Leases
+category: feature
+authors: nsoffer
+feature_name: VM Leases
+feature_modules: engine,vdsm
+feature_status: Design
+---
+
+# VM Leases
+
+
+## Overview
+
+This feature adds the ability to acquire a lease per VM on storage
+without attaching the lease to a disk. Having a VM lease, we gain two
+important capabilities; avoiding split-brain, and starting a VM on
+another host if the original host becomes unreachable.  The later
+capability will be used to improve availability of HA VMs.
+
+
+## Owner
+
+- Name: [Nir Soffer](https://github.com/nirs)
+- Email: <nsoffer@redhat.com>
+
+
+## Detailed Description
+
+
+### How it works
+
+When creating or upgrading a storage domain to version 4, vdsm will
+create a new xleases special volume. This volume will be used for
```

Comment:
You should introduce the new term. Something like:
"a new external leases special volume that will be called 'xleases'